src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/electric/machines/enchanting/AbstractEnchantmentMachine.java

```diff
@@ -55,4 +61,23 @@ protected void showEnchantmentLevelWarning(@Nonnull BlockMenu menu) {
         menu.replaceExistingItem(22, progressBar);
     }
 
+    protected boolean hasIgnoredLore(@Nonnull ItemStack itemStack) {
+        List<String> ignoredLore = ignoredLores.getValue();
+        // Skip the check if not set any ignored lore.
+        if (ignoredLore.isEmpty()) {
+            return false;
+        }
```

Comment:
Not a fan of this, the default should be to not ignore any lores, yet we have a default value.
This will have a negative impact on performance for people who don't even want to use this setting.
That's why I would rather suggest a seperate toggle setting, similar to how the level limit has one.